modules/olm-policy-fine-grained-permissions.adoc

```diff
@@ -0,0 +1,61 @@
+// Module included in the following assemblies:
+//
+// * applications/operators/olm-creating-policy.adoc
+
+[id="olm-policy-fine-grained-permissions_{context}"]
+= Fine-grained permissions
+
+OLM uses the service account specified in OperatorGroup to create or update the
+following resources related to the Operator being installed.
+
+* ClusterServiceVersion
+* Subscription
+* Secret
+* ServiceAccount
+* Service
+* ClusterRole and ClusterRoleBinding
+* Role and RoleBinding
+
+In order to confine Operators to a designated namespace, cluster administrators
+can start by granting the following permissions to the service account:
+
+[source,yaml]
+----
+kind: Role
+rules:
+- apiGroups: ["operators.coreos.com"]
+  resources: ["subscriptions", "clusterserviceversions"]
+  verbs: ["get", "create", "update", "patch"]
+- apiGroups: [""]
+  resources: ["services", "serviceaccounts"]
+  verbs: ["get", "create", "update", "patch"]
+- apiGroups: ["rbac.authorization.k8s.io"]
+  resources: ["roles", "rolebindings"]
+  verbs: ["get", "create", "update", "patch"]
+- apiGroups: ["apps"] <1>
+  resources: ["deployments"]
+  verbs: ["list", "watch", "get", "create", "update", "patch", "delete"]
+- apiGroups: [""]
+  resources: ["pods"]
+  verbs: ["list", "watch", "get", "create", "update", "patch", "delete"]
+----
+<1> Add permissions to create Deploys, Pods, and other resources.
```

Comment:
Do we need examples of how to add these permissions?

Reply:
Updated the callout to make this more clear.